inject.js

```diff
@@ -51,6 +51,11 @@ chrome.extension.sendMessage({}, function(response) {
       if (!tc.settings.rememberSpeed) {
         tc.settings.speed = 1.0;
       }
+      else
```

Comment:
If we want to enforce a minimum on saved speed, we should do so when we save the value: https://github.com/canarslan12/videospeed/blob/94dee5a802dd76739e56898c0b9b3339b621265c/inject.js#L72